examples/examples3d.jl

```diff
@@ -456,7 +456,7 @@
         scene = Scene(backgroundcolor = :black)
         scatter!(
             scene,
-            (randn(Point3f0, stars) .- 0.5) .* 10,
+            map(i-> (randn(Point3f0) .- 0.5) .* 10, 1:stars),
```

Comment:
Is this needed?  It seemed to work before...

Reply:
its fixing the depwarn for .+